distribution.pm

```diff
@@ -116,6 +116,8 @@ sub script_run ($self, $cmd, @args) {
             quiet => undef
         }, ['timeout'], @args);
 
+    die "Multiline commands are not supported:\n$cmd;\n" if $cmd =~ m/\n/;
+
```

Comment:
```suggestion
    die "Multiline commands are not supported:\n$cmd;\n" if $cmd =~ m/\n/;
```

Reply:
What is the difference in these two lines?

Reply:
The difference is the blank line removed. Also I added a comment after that which apparently became lost. So in essence please just remove the redundant blank lines.

Reply:
Personally I'm using more blank lines than @okurz does, but in our team I believe the majority doesn't like too much blank lines, so I would suggest to just remove it.

Reply:
Blank line deleted.